Remove actions menu from Social Web feed view

pfefferle opened the pull request:
Removes the actions dropdown (burger menu) containing like, boost, reply, and open original actions from the feed DataViews component. These actions were placeholders that opened the original post in a new tab, and their removal simplifies the UI while users can still click on posts to view them in the inspector panel.

## Proposed changes:
- Removed `getFeedActions` import from feed stage component
- Removed actions initialization and `useMemo` for actions
- Removed `actions` prop from DataViews component
- Rebuilt JavaScript assets

### Other information:

- [ ] Have you written new tests for your changes, if applicable?

## Testing instructions:
<!-- If you were reviewing this PR, how would you like the instructions to be presented? -->
<!-- Please include detailed testing steps, explaining how to test your change. -->
<!-- Bear in mind that context you working on is not obvious for everyone.  -->
<!-- Adding "simple" configuration steps will help reviewers to get to your PR as quickly as possible. -->
<!-- "Before / After" screenshots can also be very helpful when the change is visual. -->

* Go to '..'
*

### Changelog entry

<!-- You can optionally choose to enter a changelog entry by checking the box below and supplying data. -->
<!-- It will trigger a GitHub workflow that will create and push the entry into the branch. -->

<!-- Due to org permissions, the job may fail for PRs crated from a fork under GitHub organizations. -->
<!-- In this case, you can create entry manually with `composer changelog:add` and push it into the branch. -->

<!-- If no changelog entry is required for this PR, please add the "Skip Changelog" label. -->

-   [ ] Automatically create a changelog entry from the details below.

<details>

<summary>Changelog Entry Details</summary>

#### Significance

<!-- Choose only one -->

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

<!-- Choose only one -->

-   [ ] Added - for new features
-   [ ] Changed - for changes in existing functionality
-   [ ] Deprecated - for soon-to-be removed features
-   [ ] Removed - for now removed features
-   [ ] Fixed - for any bug fixes
-   [ ] Security - in case of vulnerabilities

#### Message

<!-- Add a changelog message here -->

</details>

pfefferle committed:
Remove actions menu from Social Web feed view

Removes the actions dropdown (burger menu) containing like, boost, reply, and open original actions from the feed DataViews component. These actions were placeholders that opened the original post in a new tab, and their removal simplifies the UI while users can still click on posts to view them in the inspector panel.

Changes:
- Remove getFeedActions import and actions initialization
- Remove actions prop from DataViews component
- Rebuild JavaScript assets

obenland commented on src/social-web/routes/feed/stage.tsx:
```diff
@@ -13,7 +13,6 @@ import { __ } from '@wordpress/i18n';
 import { Page } from '../../components/page';
 import { useFeed } from '../../hooks/use-feed';
 import { titleField, dateField, excerptField, metadataField, contentField } from '../../components/fields';
-import { getFeedActions } from './feed-actions';
```
You want to keep the feed-actions file? What would next steps be for this?

pfefferle replied:
We can remove them for now!

pfefferle committed:
Merge branch 'add/reader' into remove/burger-menu

pfefferle committed:
Remove feed actions route and add feed inspector/stage builds

Deleted src/social-web/routes/feed/feed-actions.tsx and added new build artifacts: feed-inspector.js and feed-stage.js. Updated build/social-web/index.asset.php and build/social-web/index.js to reflect these changes.

obenland commented on build/feed-inspector.js:
```diff
@@ -0,0 +1 @@
+"use strict";(globalThis.webpackChunkwordpress_activitypub=globalThis.webpackChunkwordpress_activitypub||[]).push([[99],{42:(e,t,s)=>{s.r(t),s.d(t,{default:()=>o});var i=s(427),n=s(582),r=s(723),a=s(537),c=s(486),d=s(790);const l=({html:e})=>{const t=e.replace(/\\(.)/g,"$1"),s=(0,a.decodeEntities)(t);return(0,d.jsx)("div",{dangerouslySetInnerHTML:{__html:s}})};function o({id:e,onClose:t}){const{r ...
```
This file and `build/feed-stage.js` should not exist. Could you `rm` these files and and run the build again to make sure they don't continue to get generated?

pfefferle committed:
Remove feed-inspector and feed-stage build files

Deleted the compiled JavaScript files 'feed-inspector.js' and 'feed-stage.js' from the build/social-web directory. This cleanup may be related to removing unused features or refactoring the build output.

obenland commented:
Something is not right about this PR. It seems to be missing the changes from bf08f98f73a8088af294c60de6bc3886cb4a1e0e now?

pfefferle commented:
I think it was an issue with branching and I already merged these changes with another PR :(
